### PR TITLE
chore: polish venue and form spacing

### PR DESCRIPTION
**Pull request by Sboonny**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

---

`/dashboard/chapters/{id}/new-venue`

image

- add some spacing for edit venue form

### CODE REVIEW

File: client/src/styles/Form.module.css [hunk 1]
```diff
@@ -1,5 +1,6 @@
 .form {
     display: flex;
+    margin-block: 1.5em;
```
Comment (Sboonny) [inline client/src/styles/Form.module.css:3]:
```suggestion
    margin-block: 2em;

File: client/src/modules/dashboard/Venues/components/VenueForm.tsx [hunk 2]
```diff
@@ -155,7 +155,7 @@ const VenueForm: React.FC<VenueFormProps> = (props) => {
         onSubmit={handleSubmit(onSubmit)}
         className={styles.form}
       >
-        <VStack>
+        <VStack gap={'1em'}>
```
Comment (Sboonny) [inline client/src/modules/dashboard/Venues/components/VenueForm.tsx:158]:
```suggestion
        <VStack gap={4}>

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1: 

Since all our forms have basically the same structure, can we reuse the code?  It seems a bit repetitive to have to provide custom styling for each one.

**Comment by Sboonny** [comment]
@ojeytonwilliams yeab, I can create a pull request to DRY that component and use it in the userProfile

**Pull request merged**